docs/environment_setup/Dev_Environment_Setup

```diff
@@ -1,9 +1,11 @@
 #Specific to Cent OS and RHEL
 yum update -y
-yum install -y gcc openssl-devel bzip2-devel libffi-devel zlib-devel wget make sqlite-devel
+yum install -y wget make
+yum  install -y *sqlite* --skip-broken
+yum install -y *lzma*
+yum install -y gcc openssl-devel bzip2-devel libffi-devel zlib-devel sqlite-devel xz-devel
```

Comment:
Add comments why this packages are required.

Reply:
Done